gnarly.rb

```diff
@@ -95,8 +95,7 @@ def add_ruby_version
     "\nruby \"#{RUBY_VERSION}\""
   end
 
-  copy_file "templates/.ruby-version", ".ruby-version"
-  gsub_file ".ruby-version", "__ruby_version__", RUBY_VERSION
+  # gsub_file ".ruby-version", "__ruby_version__", RUBY_VERSION
```

Comment:
[Rails](https://github.com/rails/rails/pull/30016) does this for us now, which is why this change can be removed (thanks for catching).

Is there a reason you commented it out, rather than removing?

Reply:
Perhaps #76 is related? It sounds curiously similar to a fight I had around that time that really turned into me furiously trying a bunch of things to get something to stick/pass.

Perhaps with rails relying on bootsnap, we now need to include it in our gemspec?